bin/sync.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -x
```

Comment:
rsync complains about the status of some of the files, and with -e the whole process aborts. I've removed the rm, let me know what do you think